src/main/java/twitterapp/src/TwitterAppResource.java

```diff
@@ -61,20 +56,20 @@ public Response getTimeline()
     public Response postTweet(String tweet){
     //Twitter t = twitterFactory.getSingleton();
     if (tweet.length() > MAX_LENGTH) {
-        System.out.println("Code 500: Tweet is too long, keep it within in 280 characters");
-       return Response.status(500).entity("Tweet is too long, keep it with in 280 characters").build();
+        System.out.println("Code 500: Tweet is too long, keep it within 280 characters");
+        return Response.serverError().entity("Tweet is too long, keep it within 280 characters").build();
     }
     else if(tweet.length() == 0){
         System.out.println("Code 500: No tweet entered");
-        return Response.status(500).entity("No tweet entered.").build();
+        return Response.serverError().entity("No tweet entered").build();
     }
     else {
         try {
             t.updateStatus(tweet);
         }
         catch (Exception e) {
             System.out.println("Code 500: There was a problem on the server side, please try again later.");
-            return Response.status(500).entity("There was a problem on the server side, please try again later.").build();
+            return Response.serverError().entity("There was a problem on the server side, please try again later.").build();
         }
         System.out.print("Code 200: Tweet("+tweet+") has been posted.");
         return Response.status(200).entity("Tweet("+tweet+") has been posted").build();
```

Comment:
don't use 200 and either use the Response.ok or the status code itself from the enum list